tutorials/sinon/stub.md

```diff
@@ -0,0 +1,14 @@
+With the `stub()` function, you can make testing difficult parts of your code easy.
+`stub()` substitutes the real function and returns a value you can define using `callsfake()`.
```

Comment:
"substitutes the real function with a stub, and returns the stub object."

Add an example of doing `const stub = sinon.stub(axios, 'get'); console.log(stub);` and move the rest to a separate paragraph.